cost-analyzer/values.yaml

```diff
@@ -333,7 +333,7 @@ serviceAccount:
 # These configs can also be set from the Settings page in the Kubecost product UI
 # Values in this block override config changes in the Settings UI on pod restart
 #
-# kubecostProductConfigs:
+kubecostProductConfigs:
```

Comment:
is this intentional? actually feels like a potential improvement if it doesn't cause errors